### DISCUSSION TITLE
access log for admin api?

### DISCUSSION THREAD

**Discussion by brandonbiggs** [versity/versitygw#638] [Q&A] [answered]
Hi, is there a way to enable access logs for the admin api if the admin api is running on a different port?

**Comment by benmcclelland** [comment, marked as answer]
We do not have this, but this would be a good feature request. Can you open a github issue requesting this?

**Reply by brandonbiggs** [reply]
Will do!

**Comment by benmcclelland** [comment]
admin access log added with PR #674!
